iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -115,12 +115,18 @@ public AmqpTransportSettings(TransportType transportType, uint prefetchCount, Am
 
         /// <summary>
         /// Specify client-side heartbeat interval.
+        /// The interval, in seconds, that the client establishes with the service, for sending keep alive pings.
```

Comment:
Last comma not needed
```suggestion
                /// The interval, in seconds, that the client establishes with the service for sending keep alive pings.
```

Reply:
yes this is a TimeSpan so your suggestion is good.